packages/react/src/components/DataTable/TableSelectRow.tsx

```diff
@@ -12,6 +12,57 @@ import InlineCheckbox from '../InlineCheckbox';
 import RadioButton from '../RadioButton';
 import { usePrefix } from '../../internal/usePrefix';
 
+export interface TableSelectRowProps {
+  /**
+   * Specify the aria label for the underlying input control
+   */
+  ariaLabel: string;
+
+  /**
+   * Specify whether all items are selected, or not
```

Comment:
I'm guessing that this was accidentally copied over from TableSelectAll. I decided to leave it as-is (leave it in the PropTypes docs, and to have the Typescript docs mirror the PropTypes docs) in this PR so as not to get a separate change mixed up in this Typescript conversion. Just let me know if you'd prefer that I fix this as part of this PR!

Reply:
nice catch! I do believe that's a mistake, I think it'll be fine to fix here as to not carry over more mistakes, we can rename here and in the proptypes to something like "Specify whether row is selected, or not" if you don't mind adding it 🙏🏻